pyuvdata/uvdata/mir.py

```diff
@@ -419,9 +425,13 @@ def _init_from_mir_parser(
 
             # Make sure that something weird hasn't happened with the metadata (this
             # really should never happen, only one value should exist per window).
-            assert np.allclose(spw_fsky, mir_data.sp_data["fsky"][data_mask])
-            assert np.allclose(spw_fres, mir_data.sp_data["fres"][data_mask])
-            assert np.allclose(spw_nchan, mir_data.sp_data["nch"][data_mask])
+            for val, item in zip(
+                [spw_fsky, spw_fres, spw_nchan], ["fsky", "fres", "nch"]
+            ):
+                if not np.allclose(val, mir_data.sp_data[item][data_mask]):
+                    warnings.warn(
+                        "Discrepancy in %s for win %i sb %i pol %i." % (item, *spdx)
```

Comment:
Suggest adding some guidance in the warning message on whether this is a small discrepancy vs. something critical with the data.

Or would that be caught in the checks elsewhere?